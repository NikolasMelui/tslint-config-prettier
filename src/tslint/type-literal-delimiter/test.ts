type T = {
  x: number,
  y: string
};
