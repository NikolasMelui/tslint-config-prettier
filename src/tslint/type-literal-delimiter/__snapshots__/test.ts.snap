// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (type-literal-delimiter) /src/tslint/type-literal-delimiter/test.ts[2, 12]: Expected type literal to use ';' instead of ','.
ERROR: (type-literal-delimiter) /src/tslint/type-literal-delimiter/test.ts[3, 11]: Expected type literal to use ';' to separate members.

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
type T = {
  x: number,
  y: string
};

======
type T = {
  x: number;
  y: string;
};

>>>>>> after
"
`;
